external/opensearch/src/test/java/org/apache/stormcrawler/opensearch/bolt/AbstractOpenSearchTest.java

```diff
@@ -25,7 +25,7 @@
 @Testcontainers(disabledWithoutDocker = true)
 public abstract class AbstractOpenSearchTest {
 
-    private static final String OPENSEARCH_VERSION = "2.12.0";
+    private static final String OPENSEARCH_VERSION = "2.16.0";
```

Comment:
Might be good to set this via an env variable one day, which is populated via mvn surefire, so it picks up automated changes.